src/test/java/seedu/address/logic/commands/CommandTestUtil.java

```diff
@@ -61,12 +61,12 @@
     public static final String TAG_DESC_TUTORIAL = " " + PREFIX_TAG + VALID_TAG_TUTORIAL;
 
     //    TODO once we start validating input
-    //    public static final String INVALID_NAME_DESC = " " + PREFIX_NAME + "James&"; // '&' not allowed in names
-    //    public static final String INVALID_URL_DESC = " " + PREFIX_PHONE + "911a"; // 'a' not allowed in phones
-    //    public static final String INVALID_DATETIME_DESC = " " + PREFIX_EMAIL + "bob!yahoo"; // missing '@' symbol
-    //    public static final String INVALID_MODULE_DESC = " " + PREFIX_ADDRESS; // empty string not allowed
-    //    public static final String INVALID_RECURRING_DESC = " " + PREFIX_TAG + "hubby*"; // '*' not allowed in tag
-    //    public static final String INVALID_TAG_DESC = " " + PREFIX_TAG + "hubby*"; // '*' not allowed in tags
+    public static final String INVALID_NAME_DESC = " " + PREFIX_NAME + "Lecture&"; // '&' not allowed in names
+    public static final String INVALID_URL_DESC = " " + PREFIX_URL + "911a"; // no .com
+    public static final String INVALID_DATETIME_DESC = " " + PREFIX_DATETIME
+                + "40-02-2022 1400"; // date does not exist
```

Comment:
```suggestion
    public static final String INVALID_DATETIME_DESC = " " + PREFIX_DATETIME + "40-02-2022 1400"; // date doesn't exist
```

Is it possible to fit this in one line?